game.cpp

```diff
@@ -4,19 +4,26 @@
 
 game::game()
 {
-    players.resize(2);
+  players.resize(2);
 }
 
 int game::get_player_nb() {
```

Comment:
Should the function names `get_n_players` be easier to read? Also, `get_player_nb` suggests to me I get the number (as in, index) of a player. I feel this puts me/people on the wrong foot.